NeuroID/src/main/java/com/neuroid/tracker/NeuroID.kt

```diff
@@ -120,6 +121,12 @@ class NeuroID private constructor(
         }
 
         fun getInstance(): NeuroID? = singleton
+
+        const val NID_ORIGIN_NID_SET = "nid"
+        const val NID_ORIGIN_CUSTOMER_SET = "customer"
+        const val NID_ORIGIN_CODE_FAIL = "400"
+        const val NID_ORIGIN_CODE_NID = "200"
+        const val NID_ORIGIN_CODE_CUSTOMER = "201"
```

Comment:
should these go here or in our constants file?

Reply:
yeah moved to NIDConstants, thanks!